Tools/WrapperGenerator/Templates/TypeScript/Partials/Class.mustache

```diff
@@ -33,6 +33,30 @@ export class {{ name }}
     /** @internal */
     constructor(pointer: NativePointer) {
         super(pointer);
+        const ptr = pointer.pointer;
+        const ownsPointer = this.ownsPointer;
+        {{# should_dispose }}
+        // Free WASM pointer when this object is garbage collected
+        registry.register(this, {
+            fn: (dFn, p, ownsPointer) => {
+                if (ownsPointer) {
+                    const systemsManager = Systems.SystemsManager.get();
+                    const logSystem = systemsManager.getLogSystem();
+                    if (logSystem.loggingEnabled(Systems.LogLevel.Verbose)) {
```

Comment:
We shouldn't be directly calling console.log like this in the generated code, nor manually checking the current log level. Instead, call the `LogSystem` functions.